.gitignore

```diff
@@ -5,6 +5,7 @@ ui_lovelace_minimalist/
 /themes/
 
 !custom_components/ui_lovelace_minimalist
+!custom_components/__init__.py
```

Comment:
Why are you ignoring the ini file now @stokkie90?

Reply:
It's not ignoring it! But there are some problems with the latest release of hacs itself. So I started out thinking it was caused by some changes I made.  Bit that was not the case, but I did find some.minor rename things I missed.

Reply:
Ahh, sorry, I misread it